apps/codelab/src/app/admin/feedback/feedback-message-table/feedback-message-table.component.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  Component,
+  Input,
+  ViewChild,
+  Output,
+  EventEmitter,
+  OnInit,
+  ChangeDetectionStrategy
+} from '@angular/core';
+import { MatMenuTrigger } from '@angular/material';
+import { MatTableDataSource } from '@angular/material/table';
+import { MatSort } from '@angular/material/sort';
+import { Message } from '@codelab/feedback/src/lib/message';
+
+const clearTags = (value: string) => value.replace(/<[^>]+>/g, '').replace(/Angular Codelab \/ /, '');
+const clearAllTags = (values: Message[]) =>
+  values.map((m: Message) => ({ ...m, header: clearTags(m.header || 'No header') }))
+
+@Component({
+  selector: 'codelab-feedback-message-table',
+  templateUrl: './feedback-message-table.html',
+  styleUrls: ['./feedback-message-table.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FeedbackMessageTableComponent implements OnInit {
+  @Output() closeMessage = new EventEmitter<{
+    message: Message;
+    reason: string;
+  }>();
+  @Output() takeMessage = new EventEmitter<{ message: Message }>();
+  @ViewChild(MatMenuTrigger, { static: false }) private trigger: MatMenuTrigger;
+  @ViewChild(MatSort, { static: true }) sort: MatSort;
+
+  @Input('dataSource')
+  set dataSource(values: Message[]) {
+    this._dataSource = new MatTableDataSource(clearAllTags(values));
+  }
+  _dataSource: MatTableDataSource<Message>;
```

Comment:
drop _ from the var name?

Reply:
Is this something fundamental or just a recommendation?